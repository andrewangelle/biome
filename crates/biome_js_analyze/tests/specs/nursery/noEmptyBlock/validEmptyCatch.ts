/* should not generate diagnostics */
const doSomethingTs = () => null;
try {
  doSomethingTs();
} catch(ex) {}

function fooTs(){
  try {
    doSomethingTs();
  } catch(ex) {}
}

const barTs = () => {
  try {
    doSomethingTs();
  } catch(ex) {}
}