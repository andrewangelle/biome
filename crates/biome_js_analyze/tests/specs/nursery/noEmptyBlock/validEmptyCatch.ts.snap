---
source: crates/biome_js_analyze/tests/spec_tests.rs
expression: validEmptyCatch.ts
---
# Input
```js
/* should not generate diagnostics */
const doSomethingTs = () => null;
try {
  doSomethingTs();
} catch(ex) {}

function fooTs(){
  try {
    doSomethingTs();
  } catch(ex) {}
}

const barTs = () => {
  try {
    doSomethingTs();
  } catch(ex) {}
}
```


